### ISSUE TITLE
Error if pushonce group-name contains dash "-"

### ISSUE THREAD

**Issue by uyab** [appstract/laravel-blade-directives#32]
Blade
```
@pushonce('end:popup-jenis')
<script>
    alert('once');
</script>
@endpushonce
```

`Parse error: syntax error, unexpected '=' `

Code generated:
```
<?php if(! isset($__env->__pushonce_end_popup-jenis)): $__env->__pushonce_end_popup-jenis = 1; $__env->startPush('end'); ?>
```

**Comment by ovanschie** [comment]
Fixed in 0.7.1

**Comment by fhusquinet** [comment]
Hi!

The error also happens if the stack contains a dash, for example @pushonce ('pre-scripts:my-file') will break.

Would be great to be able to do so for readability :)

**Comment by ovanschie** [comment]
@fhusquinet fixed in 1.0
